.github/workflows/post-release.yml

```diff
@@ -2,8 +2,10 @@ name: Agent Post Release
 
 on:
   workflow_dispatch:
-  release:
-    types: [published]
+  workflow_run:
+    workflows: ["Create Release"]
+    types:
+      - completed
```

Comment:
Didn't know that existed, cool.

I see one note: 

> A workflow run is triggered regardless of the result of the previous workflow.

Seems like we could get into a situation where the create release workflow fails and then this would do all the downstream stuff anyways. Wonder if there's some sort of simple check we could put into place? 

Really, only the RPM part would be problematic. Certainly easy enough to manually roll back.

Ah I see after re-reading, there is a way to gate at the job level...

` if: ${{ github.event.workflow_run.conclusion == 'success' }}`

Reply:
yea ill add this. i was testing this before lunch and forgot about it